lua/lua-core/intro-to-lua/what-to-expect.md

```diff
@@ -11,10 +11,12 @@ type: normal
 
 Congratulations on practicing with your first few lines of Lua! Sure, it may have been a simple hello statement, but **you've made progress!** 🎉
 
-This may seem underwhelming at first, but Lua can lead to very complex ideas and logic! 
+As you continue in your journey of learning Lua, you'll grasp a strong understanding and be able to flexibly apply the principles of Lua due to its ease of integration.
 
-If you're really interested, there's a Roblox Lua course that explores how to use Lua in Roblox Studio! 
+This may seem underwhelming at first, but Lua can lead to very complex ideas and logic! For instance, here's a small rotation object taught in the Roblox course!
 
-For example, here's a small rotation object taught in the Roblox course:
+![rotating-block](https://img.enkipro.com/ec104dd401c8fe092d3950f5bde6ddb1.gif)
 
-![rotating-block](https://img.enkipro.com/ec104dd401c8fe092d3950f5bde6ddb1.gif)
\ No newline at end of file
+> Note, this is strictly a course for learning the Lua language. If you'd like to specifically learn to build Roblox games, please look to the other Lua course that explores how to use Lua in Roblox Studio! 
+>
+> However, having a good understanding of Lua will aid you when working with Roblox Studio so it's recommended to learn the language!
```

Comment:
Reposting this one as it's meant to replace both of these, not just the top one:

```suggestion
> Learning Lua will help you build Roblox games and more, which is why we suggest it as your starting point.
> You can skip Lua and jump into our [Roblox Studio course](https://app.enki.com/course/roblox-studio) just know that having Lua skills is helpful for Roblox.

```